packages/eslint-config-sheriff/tsup.config.js

```diff
@@ -8,15 +8,5 @@ export default defineConfig({
   clean: true,
   noExternal: ['@sherifforg/constants', '@sherifforg/types'],
   skipNodeModulesBundle: true,
-  dts: {
-    entry: 'src/index.ts',
-    resolve: true,
-  },
-  banner: ({ format }) => {
-    if (format === 'esm')
-      return {
-        js: `import { createRequire } from 'module'; const require = createRequire(import.meta.url);`,
-      };
-    return {};
-  },
```

Comment:
can we not remove this stuff atleast for the moment? I needed this because i had some strange bug.

Reply:
The point of this is to fix the root cause those issues. They shouldn't be needed anymore. If they are, that's a bug in this PR.

Reply:
right now `dts-bundle-generator` takes **in** a `.d.ts` file and emits `out` another `.d.ts`. If you remove line 11-14, the original `.d.ts` file (generated by TSUP) will not be created, and then  `dts-bundle-generator` will fail.

Reply:
Ok so, can instead of deleting these lines, can we just comment them? We can eventually remove them later

Reply:
I can, but why? It's still there in Git, and otherwise, they'll probably never get removed.

Reply:
> It's still there in Git

Yeah i know, but for convenience.

Reply:
But do you want it convenient? It's patching the symptom, not the problem.

Reply:
I see no harm in commenting something.

Reply:
Fair enough.